tests/e2e/manifests/verify_manifest_lists.go

```diff
@@ -230,7 +230,9 @@ func getFromURLTimeoutSize(url string, timeout int, sizeOnly bool) (string, int,
 	if err != nil {
 		return "", -1, err
 	}
-	defer resp.Body.Close()
+	defer func() {
+		_ = resp.Body.Close()
+	}()
```

Comment:
why is this change needed?
does a linter complain about the lack of return value?

Reply:
I just saw a document with similar error handling in it.
https://travix.io/unhandled-errors-in-go-3f341f2704dd
I am very sorry, maybe I didn't understand.

Reply:
no problem, but given golint is not reporting this as error i don't think we should include the change.

even if `resp.Body.Close()` fails we don't need to handle the error it will return.
so `defer resp.Body.Close()` seems fine.

Reply:
Ok, I understand it.
Thank you for your patience to explain this.
:)